### ISSUE TITLE
HLE: SAW II Unimplemented function cellFsUtime

### ISSUE THREAD

**Issue by freehuntx** [RPCS3/rpcs3#1523]
Hey, i just tested the Emulator with a dump of my Saw game.
Just want to report the log.

**Comment by MyaniPT** [comment]
Try LLEing cell_Sre
Also, that's not the log, that's the console output
You can find the log at RPCS3.log in the main folder. It can be zipped and uploaded directly into GitHub

That last blank exception means the function isn't implemented yet.

**Comment by tambry** [comment]
I removed the long log. Please attach the actual log, as described by @MyaniPT.

**Comment by freehuntx** [comment]
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/174862/RPCS3.zip)
Ok, here the actual log.

**Comment by freehuntx** [comment]
After lleing cell_Sre:
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/174879/RPCS3.zip)

**Comment by MyaniPT** [comment]
```
F {PPU Thread[0x1] (main_thread)[0x0160f790]} Exception: 
(in file Emu\SysCalls\Modules\cellFs.cpp:990, in function cellFsUtime)
F {rsx::thread} Exception: Unknow window origin modifier f
(in file Emu\RSX\GCM.cpp:780, in function rsx::to_window_origin)
```
The first means the function isn't implemented yet, and you can't LLE cellFs so nothing can be done about that right now, you need to wait for someone to implement it.
The second one would be something a dev should have a look at, as I've never seen it before.

**Comment by AniLeo** [comment]
Fixed on #2289
